pymc3/backends/ndarray.py

```diff
@@ -204,7 +215,7 @@ def setup(self, draws, chain, sampler_vars=None):
         if self._stats is None:
             self._stats = []
             for sampler in sampler_vars:
-                data = dict()
+                data: Dict[str, np.ndarray] = dict()
```

Comment:
sorry, this line is too current for python 3.5...

Reply:
Oh, crumbs. I keep forgetting. I'll replace that with the corresponding type comment. My bad.